charts/tekton-pipelines/templates/kaniko/pipeline-buildonly.yaml

```diff
@@ -0,0 +1,75 @@
+{{ if .Values.kaniko.enabled }}
```

Comment:
I don't like that you performed thse changes in different way here and in pipelines. There you just added  few `if` conditions to avoid `kustomize` and `deploy` tasks, but here you have 2 separate files